python/raft/dask/common/comms.py

```diff
@@ -207,6 +260,38 @@ def local_handle(sessionId):
     return state["handle"] if "handle" in state else None
 
 
+def scheduler_state(sessionId, dask_scheduler):
+    """
+    Retrieves cuML comms state on the scheduler node, for the given sessionId,
+    creating a new session if it does not exist. If no session id is given,
+    returns the state dict for all sessions.
+
+    Parameters
+    ----------
+    sessionId : SessionId value to retrieve from the dask_scheduler instances
+    dask_scheduler : Dask Scheduler object
+
+    Returns
+    -------
+
+    session state : str
+                    session state associated with sessionId
+    """
+
+    if not hasattr(dask_scheduler, "_raft_comm_state"):
```

Comment:
This function looks like it's pretty much a copy of `worker_state`. Could we consolidate these into a function that takes the scheduler/worker instance?